src/ansys/speos/core/sensor.py

```diff
@@ -91,18 +91,33 @@ def __init__(
             self._sensor_template = ProtoSensorTemplate(
                 name=name, description=description, metadata=metadata
             )
-
             # Create local SensorInstance
             self._sensor_instance = ProtoScene.SensorInstance(
                 name=name, description=description, metadata=metadata
             )
+            self.lxp_path_number = None
         else:
             self._unique_id = sensor_instance.metadata["UniqueId"]
             self.sensor_template_link = self._project.client[sensor_instance.sensor_guid]
             # reset will fill _sensor_instance and _sensor_template from respectively project
             # (using _unique_id) and sensor_template_link
             self.reset()
 
+    @property
+    def lxp_path_number(self):
```

Comment:
this property does not align with "using get method to retrieve parameter" suggested to users. Could you please check why the get(key="lxp_properties") in the sensor class does not give result.

Reply:
i double check the _to_dict function is not grabbing class properties but just sensor instance properties
fixed with push